my_view_inc.php

```diff
@@ -342,7 +342,9 @@
 	<td class="nowrap width-13 my-buglist-id">
 		<?php
 			print_bug_link( $t_bug->id, false );
-
+			if( $t_bug->votes > 1 ) {
+				echo ' ^', $t_bug->votes;
```

Comment:
Maybe we should use '+' instead of '^', since we are using the +1 concept.  In the future, it can be +5 or -5 based on votes.